### PR TITLE
Kaptain 2.0.0 Docs legal-notices folder

### PR DESCRIPTION
**Pull request by juliangieseke**
Kaptain 2.0.0 docs "child PR" for #4309 (check its description)

### CODE REVIEW

File: pages/dkp/kaptain/2.0.0/legal-notices/index.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+layout: layout.pug
+title: Legal Notices
+beta: false
+menuWeight: 99
+excerpt: List of Third-party trademarks mentioned in the Kaptain documentation
+render: mustache
+---
+
+D2iQ&reg; and its licensors are the owners of all right, title and interest in and to D2iQ software products, the documentation, all updates, upgrades, and derivative works thereto, and all intellectual property rights therein.  D2iQ software products may additionally include third-party open source software.
```
Comment (juliangieseke) [inline pages/dkp/kaptain/2.0.0/legal-notices/index.md:10]:
```suggestion
D2iQ&reg; and its licensors are the owners of all right, title and interest in and to D2iQ software products, the documentation, all updates, upgrades, and derivative works thereto, and all intellectual property rights therein. D2iQ software products may additionally include third-party open source software.
```

**Comment by juliangieseke** [comment]
vale complaining about grammar

File: pages/dkp/kaptain/2.0.0/legal-notices/index.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+render: mustache
+---
+
+D2iQ&reg; and its licensors are the owners of all right, title and interest in and to D2iQ software products, the documentation, all updates, upgrades, and derivative works thereto, and all intellectual property rights therein. D2iQ software products may additionally include third-party open source software.
+
+# Trademarks and Registered Trademarks
+
+D2iQ, Mesosphere, Konvoy, Kommander, and Kaptain are registered trademarks of D2iQ, Inc.
+
+Amazon Web Services, AWS, AWS CloudFront, CloudFront, Elastic Compute Cloud, Amazon EC2, EC2, Elastic Load Balancer, ELB, and related graphics, logos, page headers, button icons, scripts, and service names are trademarks, registered trademarks, or trade dress of Amazon Web Services in the U.S. and/or other countries.
```
Comment (gvasquezvargas) [inline pages/dkp/kaptain/2.0.0/legal-notices/index.md:16]:
```suggestion
Amazon Web Services, AWS, AWS CloudFront, CloudFront, Elastic Compute Cloud, Amazon EC2, EC2, Elastic Load Balancer, ELB, and related graphics, logos, page headers, button icons, scripts, and service names are trademarks, registered trademarks, or trade dress of Amazon Web Services in the U.S. and other countries.
```

File: pages/dkp/kaptain/2.0.0/legal-notices/index.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+Google is a registered trademark, and Golang, Google Cloud Platform, GCP,  Google Cloud Platform service, GCP infrastructure platform, Google Cloud enterprise services, Istio, Knative, and TensorFlow open-source software library are trademarks of Google, Inc.
+
+Grafana is a trademark of Coding Instinct AB.
+
+HAProxy is a registered trademark in the United States and France.
+
+Horovod is an open-source distributed training framework for machine learning.
+
+JavaScript, MySQL, and Oracle are trademarks or registered trademarks of the Oracle Corporation and/or its affiliates.
```
Comment (gvasquezvargas) [inline pages/dkp/kaptain/2.0.0/legal-notices/index.md:50]:
```suggestion
JavaScript, MySQL, and Oracle are trademarks or registered trademarks of the Oracle Corporation and its affiliates.
```

**Pull request merged**